src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -465,6 +465,11 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
       media: MediaType.AUDIO,
       dir: Direction.DOWNSTREAM,
     },
+    audioPacketsReceivedJitter: {
```

Comment:
will legacy Chrome stat have different stat for audio jitter (I think `googJitterReceived`)? For Chrome, we always use legacy stat (unfortunately).